pkg/core/table/admin.go

```diff
@@ -11,25 +11,19 @@ import (
 
 	"github.com/open-source-firmware/go-tcg-storage/pkg/core"
 	"github.com/open-source-firmware/go-tcg-storage/pkg/core/stream"
+	"github.com/open-source-firmware/go-tcg-storage/pkg/core/uid"
 )
 
 var (
-	Admin_TPerInfoTable = TableUID{0x00, 0x00, 0x02, 0x01, 0x00, 0x00, 0x00, 0x00}
-	Admin_C_PINTable    = TableUID{0x00, 0x00, 0x00, 0x0B, 0x00, 0x00, 0x00, 0x00}
-
 	Admin_C_PIN_ColumnPIN         uint = 3
 	Admin_SP_ColumnLifeCycleState uint = 6
 
 	// TODO: This is taken from the Opal spec, not sure how to dynamically find it...
```

Comment:
You removed the code that this comment belonged to but not the comment itself, was that intended?